integration-cli/docker_cli_build_test.go

```diff
@@ -863,10 +862,10 @@ func (s *DockerSuite) TestBuildCopyWildcard(c *check.C) {
 	ctx, err := fakeContext(fmt.Sprintf(`FROM busybox
 	COPY file*.txt /tmp/
 	RUN ls /tmp/file1.txt /tmp/file2.txt
-	RUN mkdir /tmp1
+	RUN sh -c "mkdir /tmp1"
```

Comment:
It's because windows launches non-JSON commands in a dockerfile with cmd /s, so would attempt to use the mkdir from cmd. This forces it into the busybox mkdir. And yes, Windows busybox does have sh. But I don't believe it supports \, can check when in front of a computer later. Alternate solution is to use JSON syntax - six of one, half a dozen of the other...